### PR TITLE
MailChimp store info card.

### PR DESCRIPTION
**Pull request by budzanowski**
### Information

This is a part of #17434 PR that represents MailChimp Store Informations UI view. This is the third step of the setup procedure.
This is provided for review as a separate PR because the original (spike) PR is too big for the review to be comfortable. 

To test this component in action please refer to #17434.

Card needs some styling. 

image

### CODE REVIEW

File: client/extensions/woocommerce/app/settings/email/mailchimp/setup-steps/store-info.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+		const event = { target: {
+			name: 'store_locale',
+			value: e.target.value,
+		} };
+		onChange( event );
+	};
+
+	return (
+		<FormFieldset className="setup-steps__store-info-field">
+			<div>{ translate( 'Make sure that store informatin is correct. Every field is required' ) }</div>
```
Comment (timmyc) [inline client/extensions/woocommerce/app/settings/email/mailchimp/setup-steps/store-info.js]: Period at the end of this sentence.

File: client/extensions/woocommerce/app/settings/email/mailchimp/setup-steps/store-info.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+			<SettingsPaymentsLocationCurrency />
+			{ fields.map( ( item, index ) => (
+				<div key={ index }>
+					<FormLabel>
+						{ item.label }
+					</FormLabel>
+					<FormTextInput
+						name={ item.name }
+						isError={ validateFields && ! storeData[ item.name ] }
```
Comment (timmyc) [inline client/extensions/woocommerce/app/settings/email/mailchimp/setup-steps/store-info.js]: `validateFields && ! storeData[ item.name ]` is repeated 2x in this map block, maybe make an `isError` constant and use it in both spots?

File: client/extensions/woocommerce/app/settings/email/mailchimp/setup-steps/store-info.js [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+import TimeZone from 'components/timezone';
+import { translate } from 'i18n-calypso';
+
+const fields = [
+	{ name: 'store_name', label: translate( 'Store Name' ) },
+	{ name: 'store_phone', label: translate( 'Phone' ) },
+	{ name: 'admin_email', label: translate( 'Admin Email' ) },
+];
+
+const StoreInfo = ( { storeData, onChange, validateFields } ) => {
```
Comment (timmyc) [inline client/extensions/woocommerce/app/settings/email/mailchimp/setup-steps/store-info.js]: Is there any chance `storeData` could be null, and calling `store_locale` and `store_timezone` could throw errors?

Comment (budzanowski) [reply client/extensions/woocommerce/app/settings/email/mailchimp/setup-steps/store-info.js]: Changing to isReuired and making sure in top level that we always have an object.

Comment (timmyc) [reply client/extensions/woocommerce/app/settings/email/mailchimp/setup-steps/store-info.js]: Cool, maybe just to be extra safe `{ storeData = {}, onChange, validateFields }`

File: client/extensions/woocommerce/app/settings/email/mailchimp/setup-steps/store-info.js [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+				selectedZone={ storeData.store_timezone }
+				onSelect={ onTimezoneSelect } />
+		</FormFieldset>
+	);
+};
+
+StoreInfo.propTypes = {
+	onChange: PropTypes.func.isRequired,
+	storeData: PropTypes.object.isRequired,
+	validateFields: PropTypes.bool
```
Comment (timmyc) [inline client/extensions/woocommerce/app/settings/email/mailchimp/setup-steps/store-info.js]: trailing comma

**Pull request merged**